sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -33,21 +33,39 @@ appInsights
 
 export const defaultClient = appInsights.defaultClient;
 
+export function captureConsoleOutputToAppInsights() {
+  const debug = require("debug");
+
+  debug.log = (...args: any[]) => {
+    // for some reason the appinsights console.log hook doesn't seem to be firing for me (or at least
+    // it's inconsistent). For now I'll just add a hook in here and send the events myself.    
+    defaultClient.trackTrace({
+      message: util.format(...args)
+    });
+  }
+}
+
 export class SBStressTestsBase {
-  messagesSent: ServiceBusMessage[] = [];
-  messagesReceived: ServiceBusMessage[] = [];
-  trackedMessageIds: TrackedMessageIdsInfo = {};
-  snapshotTimer: NodeJS.Timer;
-  startedAt!: Date;
+  private messagesSent: ServiceBusMessage[] = [];
+  private messagesReceived: ServiceBusMessage[] = [];
```

Comment:
These are only used for reporting the count.
Let's update it to track only the count as they are too bulky?

Reply:
Yeah, let's do another PR after this (or...perhaps you'd like to? ;))